src/errors/403-en.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Forbidden - EPFL</title>
+  <meta charset="utf-8" />
+  <meta name="description" content="Forbidden - EPFL" />
+
+  {{> head }}
+
+</head>
+<body>
+  <div class="site d-flex flex-column min-vh-100">
+
+    {{> access-nav-en }}
+
+    <header class="header">
+      {{> logo }}
+    </header>
+
+    <div class="breadcrumb-container">
+      <nav aria-label="breadcrumb" class="breadcrumb-wrapper" id="breadcrumb-wrapper">
+        <ol class="breadcrumb">
+          <li class="breadcrumb-item">
+            <a href="//www.epfl.ch" title="Home" aria-label="Home">
+              <svg class="icon" aria-hidden="true">
+                <use xlink:href="#icon-home"></use>
+              </svg>
+            </a>
+          </li>
+          <li class="breadcrumb-item active" aria-current="page">Forbidden</li>
+        </ol>
+      </nav>
+    </div>
+
+    <div class="main-container">
+      <div class="w-100 pb-5">
+        <main id="main" class="content container">
+          <h1>Forbidden</h1>
+          <p>
+            You don't have permission to access on this page or ressource.
```

Comment:
Typo: ressource -> resource

Phrase: to access ~~on~~ this page

Reply:
:+1: